### PR TITLE
Add 360 degrees image viewing functionality (TAM-135)

### PR DESCRIPTION
**Pull request by urshala**
Implement 360 degrees image viewing in addition to normal images
using pannellum-react library.

panoroma

### CODE REVIEW

File: app/pages/resource/ResourcePage.js [hunk 1]
```diff
@@ -107,32 +108,63 @@ class UnconnectedResourcePage extends Component {
...
           className="app-ResourceInfo__image-button"
           onClick={() => this.handleImageClick(index)}
           type="button"
         >
           <img
-            alt={image.caption}
+            alt={isPanoromaImage ? panoromaImage.caption : image.caption}
             className="app-ResourceInfo__image"
-            src={this.getImageThumbnailUrl(image)}
+            src={isPanoromaImage ? this.getImageThumbnailUrl(panoromaImage) : this.getImageThumbnailUrl(image)}
```
Comment (wiltso) [inline app/pages/resource/ResourcePage.js]: Instead of doing 3 if/else instead of the jsx can you make one before the jsx

**Review by wiltso — COMMENTED** [review]
Could you also fix the code style in the unchanged files that the github action is warning about.

File: app/pages/resource/ResourcePage.js (hunk 1 above)
Comment (urshala) [reply app/pages/resource/ResourcePage.js]: Fixed it.

Comment (urshala) [reply app/pages/resource/ResourcePage.js]: I think its better to fix the code style in separate PR. I will fix all the style related changes there.

**Review by wiltso — APPROVED** [review]
Looks good :+1:

**Pull request merged**